x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -237,8 +245,6 @@ class AgentPolicyService {
       savedObjectType: AGENT_POLICY_SAVED_OBJECT_TYPE,
     });
 
-    this.checkTamperProtectionLicense(agentPolicy);
```

Comment:
Why we remove this here?

Reply:
since we are no longer sending in agent tamper protection on agent policy create, there is no need for a license check since agent tamper protection is the only licensed-gated feature